src/yunohost/certificate.py

```diff
@@ -810,9 +810,20 @@ def _domain_is_accessible_through_HTTP(ip, domain):
 
 
 def _name_self_CA():
-    cert = crypto.load_certificate(
-        crypto.FILETYPE_PEM, open(SELF_CA_FILE).read())
-    return cert.get_subject().CN
+    ca_conf = os.path.join(SSL_DIR, "openssl.ca.cnf")
+
+    try :
+        with open(ca_conf) as f:
+            lines = f.readlines()
+
+        for line in lines:
+            if line.startswith("commonName_default"):
+                return line.split()[2]
+    except:
```

Comment:
My try/except was aiming to catch the case where ca_conf doesn't exist, essentially... and to not block the rest of the program if something bad happens (hence the `logger.warning`). Would you like me to replace the try/except by something like `if os.path.exists(ca_conf)` ?